### PR TITLE
Added support for custom prompts

### PR DESCRIPTION
**Pull request by kiranms-newpage**
### allow customization of texts in the universal login
https://auth0.com/docs/universal-login/new-experience/text-customization-new-universal-login

The code is copied from the Pull request https://github.com/go-auth0/auth0/pull/163. I have just removed the conflicts. 
The reason is I wanted this functionality in the Terraform Provider and @gagalago has not removed the conflicts.
All the credit for this changes goes to @gagalago 

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

**Comment by vjega** [comment]
We are also looking for this merger, so that we can get rid of our fork which is several commits behind the master.

File: management/prompt_test.go [hunk 1]
```diff
@@ -2,8 +2,12 @@ package management
 
 import (
 	"gopkg.in/auth0.v5"
+	"encoding/json"
 	"gopkg.in/auth0.v5/internal/testing/expect"
+
 	"testing"
+
+	"gopkg.in/auth0.v5/internal/testing/expect"
```
Comment (gagalago) [inline management/prompt_test.go:10]: this line seems duplicated with line 6

**Review by gagalago — APPROVED** [review]

File: management/prompt_test.go (hunk 1 above)
Comment (kiranms-newpage) [reply management/prompt_test.go]: Thanks. I have removed this duplicate line.

**Review by gagalago — APPROVED** [review]

**Comment by yvovandoorn** [comment]
Hi @kiranms-newpage! What a wonderful contribution. As you noted, a lot of this was also done in https://github.com/go-auth0/auth0/pull/163, but another contribution (@arthuston) also opened up https://github.com/go-auth0/auth0/pull/220. 

Between the three, yours being the latest, could we combine the work of the three in this PR? Given that @gagalago has already signed off on this one, @arthuston - can you care to comment as well.

**Comment by arthuston** [comment]
I am fine with accepting @kiranms-newpage
<https://github.com/kiranms-newpage>'s commit, looks good!

**Comment by yvovandoorn** [comment]
@alexkappa as you reviewed #220, could you give this one a review as well (as this one is going to replace all other versions of implementation).

**Comment by kiranms-newpage** [comment]
@yvovandoorn it has been 2 months. @alexkappa has not reviewed it. 
We wanted this functionality in https://github.com/alexkappa/terraform-provider-auth0.
Any suggestions?

**Comment by arthuston** [comment]
Raise a ticket with Auth0 and see if it works.

**Comment by sergiughf** [comment]
Hello folks, apologies for the late reply on this. However this functionality was already introduced through https://github.com/go-auth0/auth0/pull/240 as such we'll be closing this down.

**Pull request closed without merging**